index.html

```diff
@@ -0,0 +1,45 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar">
+            <div class="page--logo">
+                <a class="page--link" href="#">My Logo</a>
+            </div>
+            <div class="nav--menu">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+            </div>
+        </nav>
+    </header>
+    <section class="main--content">
+        <div class="about--container">
+           <h1>I’m Charles<br>Glad to see<br> you!</h1> 
+        </div> 
+        <div class="description--container">
+           <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <p class ="supporting--text">LET’S CONNECT</p>
+        <div class="social--icons">
+            <ul>
+                <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector(2).png" alt=""></a></li>
+                <li><a href="#"><img src="images/github.png" alt=""></a></li>
+                <li><a href="#"><img src="images/peace-icon.png" alt=""></a></li>
+            </ul>  
```

Comment:
- [ ] **[OPTIONAL]:** Great job implementing the suggested changes by the previous reviewer! 💯 Please consider adding alternative text for each one of these icons, so the assistive technologies such as screen readers can have a better time browsing your site! Also, kudos ⭐  if you include your social media links so people who see your portfolio can connect with you by just clicking them! 🤓